engine/src/main/scala/cromwell/engine/EngineWorkflowDescriptor.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.engine
+
+import cromwell.backend.BackendWorkflowDescriptor
+import wdl4s._
+
+final case class EngineWorkflowDescriptor(backendDescriptor: BackendWorkflowDescriptor,
+                                          declarations: WorkflowCoercedInputs,
+                                          backendAssignments: Map[Call, String],
+                                          failureMode: WorkflowFailureMode) {
+  def id = backendDescriptor.id
+  def namespace = backendDescriptor.workflowNamespace
+  def name = namespace.workflow.unqualifiedName
+}
```

Comment:
Didn't I just see you giving thumbs downs to whitespace comments earlier today? :)